src/Nox.Lib/Exceptions/DatabaseSaveException.cs

```diff
@@ -0,0 +1,25 @@
+using System.Net;
+
+namespace Nox.Exceptions;
+
+[Serializable]
+public class DatabaseSaveException : Exception, IApplicationException
+{
+    public DatabaseSaveException() { }
+
+    public DatabaseSaveException(string message)
+        : base(message)
+    {
+    }
+
+    public DatabaseSaveException(string message, Exception inner)
+        : base(message, inner)
+    {
+    }
+
+    public HttpStatusCode? StatusCode => HttpStatusCode.InternalServerError;
```

Comment:
I'm not sure if this can actually occur, the only reason I can remember is the etag is not valid... we need to discuss if the is a server error or not?